README.asciidoc

```diff
@@ -0,0 +1,28 @@
+= Open Drive
```

Comment:
As discussed previously, I don't think it's beneficial to duplicate documentation.